NuclearData/ceNeutronData/ceNeutronCache_mod.f90

```diff
@@ -52,22 +57,22 @@ module ceNeutronCache_mod
   !!   f      -> Interpolation factor for the nuclide at energy E_tot
   !!   idx    -> Index on a nuclide grid for energy E_tot
   !!   xss    -> Cached Cross-Sections values
-  !!   needsSabInel -> Flag that tells if the nuclide is using thermal inelastic
-  !!                   scattering data
-  !!   needsSabEl   -> Flag that tells if the nuclide is using thermal elastic
-  !!                   scattering data
-  !!   needsUrr     -> Flag that tells if the nuclide is using unresolved resonance
-  !!                   probability tables
+  !!   E_maj     -> Energy at which the nuclide temperature majorant xs is stored (for TMS)
+  !!   deltakT   -> Difference between TMS material and nuclide thermal energy (for TMS)
+  !!   tempMajXS -> Temperature majorant xs value (for TMS)
+  !!   doppCorr  -> Doppler correction factor value (for TMS)
```

Comment:
No I don't think so. The thing is that we don't explain the physics or the algorithms in the User Manual, otherwise that would become messy. Maybe the best place to add an explanation is inside scatteringKernels where doppCorr is calculated?